libs/application/core/src/lib/fieldBuilders.ts

```diff
@@ -391,7 +391,7 @@ export function buildDividerField(data: {
 
 export function buildKeyValueField(data: {
   label: React.ReactNode
-  value: MaybeWithApplication<React.ReactNode>
+  value: MaybeWithApplication<React.ReactNode | React.ReactNode[]>
```

Comment:
Why is this ReactNode, if it is passed to formatText which expects FormText and returns strings?

Reply:
Yup, this should be a string. Good spot!